index.html

```diff
@@ -0,0 +1,105 @@
+                  fill="#505F79"
+                />
+              </svg>
+            </a>
+            <a
+              href="https://www.linkedin.com/in/miles-mosweu09"
+              target="_blank"
+            >
+              <svg
+                width="21"
+                height="20"
+                viewBox="0 0 21 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M14.061 3.03418C10.5722 3.03418 7.70523 5.86177 7.70523 9.37901V19.31C7.70523 19.6894 8.01611 19.9997 8.39608 19.9997H11.9885C12.3684 19.9997 12.6793 19.6894 12.6793 19.31V9.65487C12.6793 8.89625 13.3011 8.27556 14.061 8.27556C14.8209 8.27556 15.4427 8.89625 15.4427 9.65487V19.31C15.4427 19.6894 15.7536 19.9997 16.1336 19.9997H19.726C20.1059 19.9997 20.4168 19.6894 20.4168 19.31V9.37901C20.4168 5.89625 17.5843 3.03418 14.061 3.03418Z"
+                  fill="#505F79"
+                />
+                <path
+                  d="M2.90371 0C1.52202 0 0.416664 1.10345 0.416664 2.48276C0.416664 3.86207 1.52202 4.96552 2.90371 4.96552C4.2854 4.96552 5.39076 3.86207 5.39076 2.48276C5.39076 1.10345 4.2854 0 2.90371 0Z"
+                  fill="#505F79"
+                />
+                <path
+                  d="M4.69991 6.27539H1.10751C0.727545 6.27539 0.416664 6.58574 0.416664 6.96505V19.3099C0.416664 19.6892 0.727545 19.9995 1.10751 19.9995H4.69991C5.07988 19.9995 5.39076 19.6892 5.39076 19.3099V6.96505C5.39076 6.58574 5.07988 6.27539 4.69991 6.27539Z"
+                  fill="#505F79"
+                />
+              </svg>
+            </a>
+            <a href="https://github.com/Timbar09" target="_blank">
+              <svg
+                width="19"
+                height="20"
+                viewBox="0 0 19 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M18.4019 7.46114C18.4019 6.32124 18.0359 5.25043 17.3372 4.31779C17.6033 3.41969 17.6699 2.27979 17.6033 0.967185C17.5701 0.414508 17.1375 0 16.6052 0C16.3057 0 13.7771 0.0345423 12.3131 1.38169C11.0488 1.1399 9.71794 1.1399 8.42034 1.38169C6.98966 0.0345423 4.46101 0 4.12829 0C3.59594 0 3.16341 0.414508 3.13014 0.967185C3.03032 2.27979 3.13014 3.41969 3.39631 4.31779C2.6976 5.28497 2.33162 6.35579 2.33162 7.46114C2.33162 9.8791 4.12829 12.0553 6.85657 13.0915C6.75675 13.2988 6.69021 13.5406 6.62367 13.7824C3.72903 13.4715 2.36489 10.7427 2.29834 10.639C2.06544 10.1209 1.46655 9.91364 0.967475 10.19C0.468399 10.4318 0.268769 11.0535 0.534942 11.5717C0.601486 11.7444 2.43143 15.4404 6.45731 15.8549V18.9637C6.45731 19.5509 6.88984 20 7.45546 20H13.278C13.8436 20 14.2762 19.5509 14.2762 18.9637V14.8532C14.2762 14.2314 14.1431 13.6442 13.9102 13.1261C16.6052 12.0553 18.4019 9.91364 18.4019 7.46114Z"
+                  fill="#505F79"
+                />
+              </svg>
+            </a>
+          </div>
```

Comment:
- [ ] Please consider wrapping the social media list of icons in a `ul` tag instead of the generic `div` tag and that all `a` tags are wrapped in a `li` tag. Since this is a listing of social media icons it's semantically correct to wrap them in a `ul` tag.